coreos-base/update_engine/update_engine-9999.ebuild

```diff
@@ -9,7 +9,7 @@ AUTOTOOLS_AUTORECONF=1
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="2319df8546738fad1cd0ca15e1e4ca8317fce990" # v0.4.5
+	CROS_WORKON_COMMIT="7dc60831aaabda87384779424ee01c96f4ccd686" # v0.4.6
```

Comment:
The commit hash looks okay, but I'm not seeing a tag on it.

Reply:
I didn't want to push a tag until a test build finished with it.  (It's still running.)